genai/.env.example

```
@@ -3,4 +3,5 @@ LLM_API_KEY="your-api-key-here"
 LLM_API_URL=https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-flash-lite-preview-06-17:generateContent
 LLM_MODEL=gemini-2.5-flash-lite-preview-06-17
 LLM_BACKEND=google
-COURSEMGMT_URL=http://coursemgmt-service:8081
\ No newline at end of file
+COURSEMGMT_URL=http://coursemgmt-service:8081
```

```suggestion
COURSEMGMT_URL=http://coursemgmt-service:8080
```
Dies sollte auf den internen container port mappen, right?